src/core/src/OmniTileset.cpp

```diff
@@ -100,6 +100,24 @@ std::optional<CesiumIonClient::Token> OmniTileset::getIonAccessToken() const {
     return t;
 }
 
+std::string OmniTileset::getIonApiUrl() const {
+    auto tileset = UsdUtil::getCesiumTileset(_tilesetPath);
+
+    pxr::SdfPathVector targets;
+    tileset.GetIonServerBindingRel().GetForwardedTargets(&targets);
```

Comment:
Note to self - check if backwards compatibility is possible here. If there's no binding, create one that points to the default ion server.

Reply:
Will handle this in a different PR